sources/packages/forms/src/form-definitions/institutionprofile.json

```diff
@@ -395,7 +396,7 @@
           "validate": {
             "required": true,
             "minLength": "",
-            "maxLength": 100,
+            "maxLength": 250,
```

Comment:
Nice catch updating this form also.
Can we have an AC added to the ticket to make it clear that the update process by the Ministry is also a scenario to be tested by the business?
@JasonCTang FYI